scripts/verify-replace.sh

```diff
@@ -5,6 +5,7 @@ GH_BASE_URL_KS=https://github.com/kubesaw/
 GH_BASE_URL_CRT=https://github.com/codeready-toolchain/
 declare -a REPOS=("${GH_BASE_URL_KS}ksctl" "${GH_BASE_URL_CRT}host-operator" "${GH_BASE_URL_CRT}member-operator" "${GH_BASE_URL_CRT}registration-service" "${GH_BASE_URL_CRT}toolchain-e2e")
 C_PATH=${PWD}
+API_REPLACE=$(grep 'replace' go.mod)
```

Comment:
You cannot simply grep `go.mod` for any replace clause - there could be multiple of them and in two formats.
During our 1-1, I pointed you to the way of checking replace clauses in toolchain-cicd repo:
https://github.com/codeready-toolchain/toolchain-cicd/blob/d5e8b3cdde0dd75c1de1735c22d2fb8adf2d37f1/gomod-check/action.yml#L24
is there a reason why you didn't use it?

Reply:
```
go mod edit -json | jq '.Replace'
```

thank me later :D

Reply:
@MatousJobanek i did try using that, the main concern for me was to construct a replace of api via script and add it to other repos after fetching it through the above command. somehow i thought of simplifying it and i just forgot about 2nd type of format of replace . i presumed, in toolchain-common, we will have only api replace clause atleast in the PRs where we require this check.🤦🏻‍♀️

Reply:
@drpaneas thanks for this, checking if i can use that 👍